website/docs/reference/node-selection/defer.md

```diff
@@ -17,6 +17,24 @@ Defer requires that a manifest from a previous dbt invocation be passed to the `
 
 ### Usage
 
+<VersionBlock firstVersion="1.2">
+
+```shell
+$ dbt run --select [...] --defer --state path/to/artifacts
+$ dbt test --select [...] --defer --state path/to/artifacts
+```
+
+</VersionBlock>
+
+<VersionBlock firstVersion="0.21" lastVersion="1.0">
+
+```shell
+$ dbt run --models [...] --defer --state path/to/artifacts
+$ dbt test --models [...] --defer --state path/to/artifacts
+```
+
+</VersionBlock>
+
```

Comment:
At the moment, the `firstVersion` is required and the `lastVersion` is optional. Rather than having to set `firstVersion="0.0"`, it would be a better experience to be able to exclude the firstVersion prop altogether. That should definitely be possible, so I will revisit this to adjust